README.md

```diff
@@ -2,8 +2,16 @@
 
 [![Build Status](https://travis-ci.org/puppetlabs/trapperkeeper-status.svg)](https://travis-ci.org/puppetlabs/trapperkeeper-status)
 
-A trapperkeeper service for getting the status of other trapperkeeper
-services.
+A Trapperkeeper service that provides a web endpoint for getting status
+information about a running Trapperkeeper application.
```

Comment:
I don't think so; there will only be one TK application involved... there may be multiple TK services, but only one application.